tests/topology/expected/vrf-loopback.yml

```diff
@@ -8,6 +8,7 @@ nodes:
   r1:
     af:
       ipv4: true
+      vpnv4: true
```

Comment:
Doesn't this point at a missing test case? The extra loopbacks should get distributed using VPNv4, if it wasn't doing that before then that's a bug (?)

Reply:
This was an edge case -- a VRF that contained only a loopback interface -- and the difference between before and now is that now the loopback is created before BGP post_transform hook, so the extra AF is set.

In theory, that could have caused a problem in a pretty contrived MPLS/VPN scenario, but now we fixed it anyway ;)

As for missing test case, you might claim we were sloppy inspecting the test results of that test case 🤷‍♂️